src/wasm-validator.h

```diff
@@ -28,7 +28,8 @@ namespace wasm {
 
 struct WasmValidator : public PostWalker<WasmValidator, Visitor<WasmValidator>> {
   bool valid = true;
-  bool validateWebConstraints = false;
+  bool validateWeb = false;
+  bool validateGlobally = true;
```

Comment:
Can there be some comment about what `validateGlobally` does and/or is for? Is it intended to be temporary?